### DISCUSSION TITLE
Einstellungen mit BluePalm Speicher und IR Zähler

### DISCUSSION THREAD

**Discussion by 3joern** [hoylabs/OpenDTU-OnBattery#803] [Q&A]
Moin Zusammen,
verdammt, ich dachte ich bekomme das alleine hin, aber ich steck zu wenig drin.
Habe 2 Pannels (a 390W) einen HM600, ein Hitchi RI Leesekopf und nun openDTUonBattery, in dem der Zähler auch schon erkannt und gezeigt wird. Aber bei den Einstellungen von den Dynamic Power limiter Settings komm ich nicht weiter. Inzwischen zeigt der WR nur noch Gelbe fehler an, wenn ich den WR neu starte geht kurz, dann wieder gelb.
Als Speicher habe ich den [BluePalm](https://media.hornbach.de/hb/ownersmanual/as.77760358.pdf), der Zwischen Pannels und WR hängt. Habe ihn ohne an der WR zu hängen vorher mal auf 100% geladen und nun alles zusammengesteckt. 
Aber bei wenig Sonne, also wenn der Zähler hoch geht, kommt nichts aus dem Speicher.  Gefühlt werden die Daten vom Zähle rnicht angewendet. Vorher hat der Speicher einfach auf durchzu geschlaten und nur Solarstrom durchgelassen. 
Hätte da jemand eine Idee, wie ich das sinn voll als Nulleinspeisung einstelle?

Hier der Konsolentext:
https://pastebin.com/3NXPwpZx

Einstellungen: 
![Bildschirmfoto 2024-03-25 um 12 18 13](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/177b68e3-4c46-4235-86bf-220c328a86f6)
![Bildschirmfoto 2024-03-25 um 12 14 06](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/053406c7-c3ab-4011-ba43-6cf3dd0aba79)
![Bildschirmfoto 2024-03-25 um 12 14 12](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/8e4c512c-d9ec-4bad-9826-6e2f3a790931)

![IMG_6815](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/464c16fe-c5cb-4a97-9aca-678d98a04049)
![IMG_6816](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/45191d29-7bb6-4229-9bfc-820dc843dcec)

**Comment by spcqike** [comment]
Deine Stopp Spannung muss kleiner sein als deine Batterie nennspannung. 49V Stopp ist schon echt hoch. Für eine Batterie die im screen nur 46V zeigt und im log sogar mal nur 41V.

**Reply by 3joern** [reply]
Welche Spannung wäre angebracht? bzw wo bekomme ich das raus? habe hier einige Zahlen, welche soll es sein?

![Bildschirmfoto 2024-03-25 um 12 28 08](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/cd62a1c6-ec94-433c-808c-99307f820c7a)
![Bildschirmfoto 2024-03-25 um 12 28 31](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/0ae908d6-e592-4ede-8a48-2e689b62e1e5)

Sooo? 

![Bildschirmfoto 2024-03-25 um 12 34 49](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/372aaabe-22db-4a74-acf1-d08924f7f9f8)

was ist mit der Solar Passthrough  Spannung? das sind beide Panele zusammen? bei voller Last sind es gerade 73,2 Volt

![Bildschirmfoto 2024-03-25 um 12 39 06](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/580b9f1f-b099-4cca-8d9a-3e2552f3613e)

**Reply by spcqike** [reply]
Ist halt die Frage ob du echt die batteriespannung am Ausgang hast, oder ob das Ding solarmodule „simuliert“ und die Spannung einfach selbst erzeugt. Ja, es muss ja nicht mal eine ~48V Batterie sein. Es kann ja eine 24V Batterie sein und ein leistungs stepup regelt einfach hoch. 

Nimm erstmal 43V an, in der Hoffnung, dass es ne echte batteriespannung ist die dir da am Ausgang gegeben wird. Und beobachte ob sie stark schwankt bzw. ob die Spannung bei hohen SoC gering ist. (Im abzuschätzen, ob du da wirklich die Batterie Spannung bekommst. Wenn nämlich nicht, wäre es eh egal. Dann kannst du stoppen wie du willst, die echte Batterie kannst du damit nicht schützen) 

ein Nein. Du interpretierst die Spannungen falsch. 

„Fang an die Batterie ab 48V zu entladen, und entlade runter bis 43V“. Start 48, Stopp 43.

**Reply by 3joern** [reply]
Vielen Dank und Sorry, ich habe gerade ein paar Screenshots noch aktualisiert. 
DC Start 48, Stopp 43 habe ich jetzt erstmal drin.

Was wäre bei Solar Passthrough Sinnvoll?

**Reply by spcqike** [reply]
> bei voller Last sind es gerade 73,2 Volt

sind die Module denn in Reihe? Das wäre für deine Batterie und auch für den wechselrichter (sollte die Batterie die PV spannung 1:1 weiterleiten) äußerst schlecht. 

> Was wäre bei Solar Passthrough Sinnvoll?

geht das mit dem setup überhaupt? Braucht es dafür nicht einen Victron MPPT?

**Comment by 3joern** [comment]
Sorry, kam jetzt erst zum Testen und habe jetzt nur mal die Batterie am Wechelrichter. PV Module sind nicht in der Batterie eingesteckt (die 73,2V waren aber beide addiert). WR akzeptiert das immer noch nicht hier die neue Konsole: https://pastebin.com/jAChiAwU

Einstellungen: 
![Bildschirmfoto 2024-03-25 um 18 56 02](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/6f2a16b2-e0cc-41bb-8000-36ba9ec8d8df)

![Bildschirmfoto 2024-03-25 um 18 54 57](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/eb6dd139-7cec-4be3-92d3-c03e6c700adc)
![Bildschirmfoto 2024-03-25 um 18 55 03](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/e11b8a03-4098-4ecf-9a0c-2600439f20a7)

> geht das mit dem setup überhaupt? Braucht es dafür nicht einen Victron MPPT?

Denke nicht, was soll denn dann drin stehen, wenn da schon eigene MPPTs verbaut sind?

[Das ist diese Kiste](https://media.hornbach.de/hb/ownersmanual/as.77760358.pdf)

Ps. schalte ich den Dynamic Power limiter Settings aus, legt der gleich los und speichert die vollen 612W raus, achtet dann eben nicht auf den tasmota IR Kopf

**Reply by spcqike** [reply]
> die 73,2V waren aber beide addiert

Ich hoffe bei den gezeigten specs immer noch, dass du die Module nicht in Serie geschaltet hast. 

Der DPL wird nicht gehen, weil deine startspannung (48V) nicht erreicht ist. Der Akku liefert aktuell scheinbar nur 46.5V.

**Reply by 3joern** [reply]
> Ich hoffe bei den gezeigten specs immer noch, dass du die Module nicht in Serie geschaltet hast.
Jedes Modul geht einzeln in einen PV EIngang am Speicher. Da schaue ich extra drauf. 

> Der DPL wird nicht gehen, weil deine startspannung (48V) nicht erreicht ist. Der Akku liefert aktuell scheinbar nur 46.5V.
Das war hilfreich, ich habe das mal auf 
DC Spannung - Start: 46V
DC Spannung - Stop: 42V
gestellt, nun scheint alles zu laufen (wobei ich Sorge habe, dass die geratenen Zahlen nicht stimmen, sobald die Spannung ausserhalb dieses Bereichs ist),. 
Was ist aber, wenn Tagsüber der Speicher voll ist und ich gerne fürs Büro den Strom durchreichen wollen würde? geht das so?

Aber das sieht schonmal gut aus.
![Bildschirmfoto 2024-03-25 um 19 31 31](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/de8e5116-56a4-4552-8cc0-670930c613b3)

So gemacht (shiiii, gerade mal Solar-Passthrough aktiviert und schon wieder gelb, nun wieder deaktiviert, dann fallen die unteren Felder weg, dann läufts aber wie erwartet):
![Bildschirmfoto 2024-03-25 um 19 33 56](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/e4873e71-d98c-4424-9cdf-d91cd9e9b26f)

**Reply by 3joern** [reply]
Danke übrigens für Dein super Support ;)
ohne Dich würd ich echt dumm da stehen, langsam verstehe ich die Funktion besser.

**Comment by 3joern** [comment]
Oh, nachdem es nun mit 
DC Spannung - Start: 46V
DC Spannung - Stop: 42V
Zu klappen scheint, ist da aber doch wohl etwas zu viel Spiel und das schaltet sich hin und wieder doch kurz ab, siehe Meldungen:
![Bildschirmfoto 2024-03-25 um 19 42 50](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/145d172e-d848-4c1e-8ee1-c35990655d56)

**Reply by spcqike** [reply]
Bitte beobachte da mal die Spannungen in live view oder erstelle einen log. 

Icu vermute, dass die Spannung, die die Batterie ausgibt, schwankt oder stark einbricht. Wie gesagt, ich bin mir noch nicht ganz sicher, dass die ausgegebene Spannung auch wirklich einer Batterie entspringt, oder ob sie durch stepupwandler, die ein Solar Modul simulieren wollen, künstlich erzeugt wird und dementsprechend schwankt.

**Reply by 3joern** [reply]
Hui jetzt bin ich echt verwirrt, WR ist Offline? hatte ich so auch noch nie

https://pastebin.com/ctPzjrDs

![Bildschirmfoto 2024-03-25 um 20 09 56](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/4a55cff0-23ec-4a1c-9821-0c32126c429b)

Gehe jetzt essen, das macht einen ja kirre, Danke und einen schönen Abend.

**Reply by spcqike** [reply]
Fragst du den inverter denn nachts ab? Ist der Haken in den wechselrichter Einstellungen gesetzt?

**Reply by 3joern** [reply]
🤦‍♂️ ok, das hätte ich eigentlich wissen sollen, Jupp war Nachts immer aus. Na dann, danke. Werde mir das morgen nochmals genauer anschauen.

**Reply by schlimmchen** [reply]
Hast du da keinen entsprechenden Hinweis gesehen bei den DPL Einstellungen? Welche Version der Firmware nutzt du?

Achja, die Screenshots sind aus einer älteren Version. Die neue hätte dir einen Hinweis gegeben...

**Comment by 3joern** [comment]
Moin, so habe das Update gemacht und da sind nun ein paar Einstellungen anders, habe rumprobiert und wenn ich nach wie vor 
DC Spannung - Start: 46V
DC Spannung - Stop: 42V
eingebe, wird es wieder gelb. Habe nun bei beiden 16V und es läuft, was aber sehr irritierend ist, die Batterie App sagt, da gehen 224W an den Wechselrichter, aber zur gleichen Zeit sagt der Wechselrichter, da sind nur 111,7W, woran kann das liegen und Netzbezug ist um die 100W? also nicht wirklich Null !? 

https://pastebin.com/sKWFjp2U

![Bildschirmfoto 2024-03-26 um 11 07 46](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/7144980c-1602-4c27-9174-21ba2ba7dea6)

![Bildschirmfoto 2024-03-26 um 11 11 48](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/3e0a83ac-ef3e-489e-802f-e5941ef0a4dc)

![IMG_6818](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/211de01f-064e-44e7-8436-f54395922ad9)


Ps. Sorry, wenn ich Euch langsam auf den Zeiger gehe, ich finde das Thema unglaublich Spannend, aber bin in einer absoluten Lernphase 

PPS. kann man Screenshots auch kleiner einbinden? der vom Smartphone zerschiesst doch sehr den Lesefluss ;)

**Reply by schlimmchen** [reply]
Warum hast du am zweiten Eingang keine Leistung? Ist der nicht angeschlossen?

Langsam wirds anstrengend das immer und immer wieder zu wiederholen. Ich frag mich, wo wir das wie aufschreiben müssen... Das Limit wird vom Hoymiles gleichmäßig auf alle MPPT aufgeteilt. Wenn du am zweiten Eingang keinen Saft hast, dann kommt in Summe aus deinem HM-600 nur "die Hälfte".

**Reply by 3joern** [reply]
> Warum hast du am zweiten Eingang keine Leistung? Ist der nicht angeschlossen?

![Bildschirmfoto 2024-03-26 um 13 08 02](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/2ef7d6bb-f4a2-4a1f-9232-f45b0641abce)

Vorher hat der eine Eingang eingespeist und der andere den Akku geladen. Alternative einstellung am Akku wäre erst beide laden und dann einspeisen/durchreichen. Da ich aber Tagsüber im Homeoffice bin würd ich schon gerne die Sonne schon während dem Laden nutzen. Aktuell gehts ja, war nur verwundert, warum die Batterie App andere Werte als der WR anzeigt. 

> Langsam wirds anstrengend das immer und immer wieder zu wiederholen.

wirklich Sorry, ich habe nirgends eine Dokumentation dazu gefunden, die mir das so erklärt (vor allem das man es als nicht Entwickler/Ingenieur versteht) und um mir das einfach zusammen zu reimen, fehlt mir "noch" die Expertise.

> Das Limit wird vom Hoymiles gleichmäßig auf alle MPPT aufgeteilt. Wenn du am zweiten Eingang keinen Saft hast, dann kommt in Summe aus deinem HM-600 nur "die Hälfte".

Ah ok, so versteh ich das. Hm nur dumm, dass der Speicher switcht. Beim laden wird, wie gesagt, einer zum Wechselrichter durchgereicht und der andere läd die Batterie. Beim Entladen durch beide wieder raus... da könnte ich aber auch nur einen aktivieren... scheint mir nach Deiner Summen Aussage irgendwie sinnvoll, wäre dann aber auf 300W begrenzt.

So ist ja aktuell aufgebaut

![Bildschirmfoto 2024-03-26 um 13 17 45](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/fa0d5afd-1aa5-4e3f-a060-5ce5ae80e35e)

Dann wäre das wohl dieses Szenario, was Du meinst, was für den HM600 sinnvoller wäre (oder eben über die App)?
![Bildschirmfoto 2024-03-26 um 13 17 55](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/3b8b11c3-6d70-4c76-8c20-f70344fb9cc5)

**Reply by spcqike** [reply]
Dein Speicher scheint nicht wirklich clever oder smart zu sein. 

Warum bitte wird ein Paneel komplett durchgereicht, das zweite zum Laden genutzt? Warum kann er nicht einfach „alles“ durchreichen und das, was nicht abgenommen wird, in den Akku stecken? 

Und warum meldet die App, dass >200W an Ausgang 2 fließen, wenn der HM nur 120W nimmt? Die Differenz, also die 80W, könnten ebenfalls gespeichert werden. 

Ich würde an deiner Stelle nicht mischen. Also nicht 1 Paneel an den HM und 1 an die Batterie. 

Der deaktivierte Ausgang liefert leider noch immer fast 5W. Das erkennt die openDTU. Wäre die Leistung dort geringer, würde die openDTU dein gewünschtes Limit verdoppeln, sodass der HM auf einem Eingang eben bis zu 300W sauber erzeugen kann. 
wobei ich grade sehe, dass der Eingang nur 1.4V hat. Bei über 3A. 

@schlimmchen 
> Wenn du am zweiten Eingang keinen Saft hast, dann kommt in Summe aus deinem HM-600 nur "die Hälfte".

er hat ja etwas saft. Zu viel, als dass die Skalierung hoch regelt :) 
Macht es Sinn, bei der Skalierung zusätzlich zur Leistung auch die Spannung am Eingang zu berücksichtigen? Theoretisch sollte der hoymiles an dem Eingang ja gar nichts liefern, da die 22V MPPT start Spannung nicht erreicht sind. Und da er unter 16V ist sollte der tracker eigentlich stoppen. Ich vermute dort fließt, aufgrund der geteilten Masse, einfach etwas über den falschen Ausgang zurück, oder?

**Reply by schlimmchen** [reply]
> wirklich Sorry, ich habe nirgends eine Dokumentation dazu gefunden

Das geht auf unsere Kappe, wir sollten es wenigstens irgendwo hinschreiben :wink:

Ich check leider nicht dieser BluePalm macht... Ich hab nur begriffen, dass du nur Leistung auf einem Input hast.

**Comment by 3joern** [comment]
> Das geht auf unsere Kappe, wir sollten es wenigstens irgendwo hinschreiben 😉

Wenn Ihr support von einem sehr erfahrenen UI/UX Designer braucht, sagt bescheid, optisch und mit der Usability kann ich Euch da gerne unterstützen (das es auch so einer wie ich versteht  😉)


Aktuell läuft es mit folgenden Einstellung ganz gut, auch mit Decken abgedeckt um zu sehen, wann dann der Speicher abgibt und ob es da Probleme mit der Spannung gibt - sehr Spannend das 🤪. Hab einiges durchgetestet und immer auf die Spannung geschaut, mit im Hinterkopf die Summierung - obwohl in den Einstellungen "Eingang für Spannungsmessung" nur ein Eingang gewählt werden kann, etwas irritieren. Ich teste das jetzt mal die Tage und wenns Mist ist, geht die Kiste direkt wieder zurück zum Hornbach. Dachte bei 750Öro und 2240W könnte mans wenigstens mal versuchen.

![Bildschirmfoto 2024-03-26 um 16 33 21](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/4486850b-20bf-475f-bed4-783bae497b42)

![Bildschirmfoto 2024-03-26 um 16 33 45](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/d25635df-7511-49b7-8e6f-7526658f39b9)

**Reply by 3joern** [reply]
Fuuuuu... gerade meinte ich noch, nachdem alles ne Stunde richtig gut lief, och deaktiviere ich mal überall die Protokollierung. Vielleicht hat er dann etwas mehr Ressourcen. Zack, wieder im Standby und egal wo ich was einstelle, geht nicht zurück. Langsam hab ich keine Lust mehr.
Tante Edit: nach hin und her testen, scheint das an der Batterie App zuliegen, wenn ich da hin und her switche, zwischen beide Ausgänge aus, dann einem Ausgang und dann beide online schalte, kriegt sich der WR wieder ein. Dabei die App die Daten immer wieder aktualisieren lassen. Sauber ist das ganze nicht 🤦‍♂️

**Reply by schlimmchen** [reply]
Dein unteres Leistungslimit ist zu niedrig. Die Regelung der Hoymiles scheint zu schwingen bei niedrigen Leistungslimits, und dann schalten sie sich ab und nur ein Neustart hilft. Kann das bei dir der Fall sein?

**Comment by Manos1966** [comment]
> Langsam wirds anstrengend das immer und immer wieder zu wiederholen. Ich frag mich, wo wir das wie aufschreiben müssen... Das Limit wird vom Hoymiles gleichmäßig auf alle MPPT aufgeteilt. Wenn du am zweiten Eingang keinen Saft hast, dann kommt in Summe aus deinem HM-600 nur "die Hälfte".

Ich kann mich erinnern ich habe Mal diese Frage an jemanden gestellt, er wollte aber mir nicht helfen.... kann mich aber an Namen und Personen nicht erinnern,,, 👻🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣


"Etwas" haben wir schon, aber ich habe keine Info ueber die "Erkennung der aktiven MPPTs"
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/FAQ

**Reply by spcqike** [reply]
Die Funktion hier ab Zeile 643

https://github.com/helgeerbe/OpenDTU-OnBattery/blob/91a0992964b53dc6198dc53bdc5a169b7bede2b7/src/PowerLimiter.cpp#L643

Kurz um: 
Die Skalierung geht nur bei inverter, wo jeder MPPT einen eigenen Eingang hat. Nur wenn die gesamt Leistung > Anzahl der Eingänge * 10. (Beim HM-800 also >20W, beim hms-1600>40W).  Eingänge, die dann mehr als 2W liefern sind „aktiv“. Die anderen inaktiv. Am Ende wird entsprechend skaliert.

**Reply by Manos1966** [reply]
@spcqike und beim HM-1200/1500 (zwei MPPTs vier Eingänge )?

**Reply by schlimmchen** [reply]
Da wird nicht mehr skaliert, weil der Code erst hinreichend intelligent werden müsste.

**Reply by Manos1966** [reply]
OK, ich habe etwas addiert
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/FAQ

Dann kann ich also folgendes addieren: 
- Wechselrichter, die einen MPPT pro Eingang haben, werden skaliert (unter der Voraussetzungen die @spcqike erwaehnt hat)
- Bei Wechselrichter dessen MPPTs mehr als einen Eingang steuern, findet kein Skalierungsversuch statt

passt es?

**Reply by spcqike** [reply]
Ja ich denke das sollte passen. (:

**Reply by Manos1966** [reply]
> Langsam wirds anstrengend das immer und immer wieder zu wiederholen. Ich frag mich, wo wir das wie aufschreiben müssen.

Ich habe es auf der Wiki-Seite des Dynamic-Power-Limiters auch addiert da solche Fragen nur kommen, wenn die Werte des Power Limiters "nicht richtig gefolgt" werden...
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter

Ihr (@schlimmchen @spcqike ) könnt es beliebig anpassen....

**Comment by 3joern** [comment]
Sorry, weiter gehts. Aktuell hab ich alles stabil zum laufen gebracht, bin aber sehr verwirrt über das was der HM600 macht. In der BatterieApp heisst es werden ~160W abgegeben, im WR werden auf beiden Strings jeweils 77-78W abgegeben, aber in der Großen Übersicht stehen da nur 75W. Zählerdaten stimmen. Was passiert mit den anderen 50%? das ist mir heute schon bei voller Sonne aufgefallen, dass beim bypass 245W in der App standen, im Wechselrichter aber nur 120W bei Inverter Gesamtleistung.  Oberes Leistungslimit hab ich gerade auf 150W begrenzt um zu schauen wie sich das auswirkt, den Tag über war es auf 350W begrenzt. hat jemand eine Idee? Oh gerade sehe ich 45% Wirkungsgrad, was läuft denn da schief?

Konsole dazu
https://pastebin.com/MMJezc7a

![Bildschirmfoto 2024-03-28 um 16 38 42](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/7397b523-22e2-4c74-b9c9-d8e97c0e289f)

**Reply by spcqike** [reply]
Das mit dem Wirkungsgrad ist merkwürdig. Das kenne ich bisher nur wenn die wechselrichter unrund laufen weil die Spannung oder irgendwas kurzzeitig nicht sauber war. Zu tief. Oder kurz weg. Oder was auch immer. Das solltest du beobachten. 

Dass die Ausgangsleistung der App nicht der eingsngsleistung des Eingangs entspricht, erkläre ich mir damit, dass der Ausgang „bis zu x W“ liefert. Genau das, was dort an Eingang 2 anliegt und vom Paneel kommt. 
Der wechselrichter läuft aber mit einem Limit (138W) und zieht damit nicht mehr als ~70W pro Eingang. 
deine Batterie wird schlicht und einfach nicht genau messen, welche Leistung am Ausgang wirklich fließt.

**Reply by 3joern** [reply]
Danke fürs schnelle Feedback. Sorry, weil da einiges am 2. Screenshot nicht gestimmt hat, hab ich den wieder rausgenommen. Werde das weiter beobachten. Ich habe über den Tag hinweg je nachdem wie die Wolken fliegen einige Fehlermeldungen (WR kurz gelb) schaltet dann selbstständig wieder auf blau. Ist das Problematisch? möchte das Ding ja nicht vorzeitig grillen ;)

Hab aber die längerfristigen Abschaltungen nur damit in den Griff bekommen, dass ich den 
Start Schwellenwert = 20
Stop Schwellenwert = 19
gestetzt habe. Hab ich was höheres gewählt ist der WR sofort sobald ne Wolke kam oder voll die Sonne schien in dauer Standby gerutscht. So läuft er nun halbwegs stabil, bzw pendelt sich wieder ein und ist nach ein paar sek wieder blau.

**Reply by Manos1966** [reply]
@3joern 
Meine Erfahrung: Siehst du Effizienz zwischen 50% - 65%, es ist Zeit den Wechselrichter neu zu starten.
Das Problem hatte ich gelegentlich mit den PowerQueen Batterien 2x25V = 50V Konfiguration ohne Balancing.
Kein einziges Mal seitdem ich die Pylontech habe.

**Reply by spcqike** [reply]
> Ich habe über den Tag hinweg je nachdem wie die Wolken fliegen einige Fehlermeldungen (WR kurz gelb) schaltet dann selbstständig wieder auf blau. Ist das Problematisch? möchte das Ding ja nicht vorzeitig grillen ;)

das passiert bei allen wechselrichtern, die an solar Paneelen hängen. Die Verschattung an sich ist kein Problem. Kritisch sehe ich nur deine Batterie. Da wir nicht wissen, wie genau sie funktioniert und wie das durchreichen der PV Spannung bzw. Akku Spannung geht. Schaltet der den Ausgang dann kurzfristig ab? Fällt die Spannung auf <16V? Welche Spannung hat die Batterie am Ende, wenn die mal durchgereicht wird? Sind die >45V im Bild überhaupt vom Solar Paneel? (Was für Paneele hast du?)

der MPPT im hoymiles startet erst ab 22V, geht dann aber zwischen 16-60V. Fällt der Eingang also <16 geht der wechselrichter aus und meldet gelb. 

Nachtrag
Deine Spannungen kann ich auch nicht nachvollziehen. Mein HM-600 hängt an 2 Paneelen mit je 144halbzellen. Selbst bei <10W pro Eingang hab ich noch >30V Spannung. Warum du also als Start und Stopp Spannung 20/19 eintragen musst, erschließt sich mir nicht. Die echte PV Spannung ist recht schnell >30V und bleibt da, selbst bis 5W runter, eigentlich dauerhaft weit drüber. 

<img width="375" alt="image" src="https://github.com/helgeerbe/OpenDTU-OnBattery/assets/50098985/b03b1ade-526a-4e89-85ce-c6a161c16583">

**Reply by 3joern** [reply]
Hi, sorry wenn ich da nochmal nachhake, mir ist inzwischen aufgefallen, dass der WR immer in Standby geht, wenn ich zB den Wasserkocher oder die Anlage an oder aus mache. Kann das sein? das sobald heftige Verbraucher Strom ziehen, dass dann durch den extremen Unterschied im PowerLimiter der WR kurz überfordert ist? (aber egal ob durch Batterie oder wenn PV durchschleift - ausser ich deaktiviere den Power Limiter). Wie gesagt hab ich einen sehr niedigen Start Stop Schwellenwert, dass es überhaupt mal rund läuft. Die Batterie sollte eine Spannung von 42-50,2V haben (laut Zettel), die Pannels zwischen 35 und 37V (so wie ich das oben aus dem Datenblatt rauslese). Evt wäre es super, wenn Ihr mir kurz sagen könntet, wie das mit der Start Stop Spannung zu verstehen ist, ich bin zu fachfremd um das an meien Parameter anzupassen, wenn ich nicht kapiere, wie das funktioniert.    
In den Facts lese ich:

> Batterienutzung Start/Stop-Schwellwert: Geben Sie die Spannung ein, die laut Batterie Handbuch den gleichen Ladestand von Akku SOC – Start/Stop bedeutet (in diesem Beispiel, für unsere Batterie, 51V sind ein Ladestand von 75%, sowie 46V sind Ladestand von 20%).

Dann wäre in meinem Falle 42V nahe der 0% (wobei ich in der App die Netzgekoppelte DoD auf 90% - also 10% Restkapazität soll im Akku bleiben, eingestellt habe) und 50,2V wäre dann 100% voll? Also sollte es zB damit richtig sein? 
Start Schwellenwert = 48,56V = 80%
Stop Schwellenwert =  42,82V = 10%

**Comment by 3joern** [comment]
> Meine Erfahrung: Siehst du Effizienz zwischen 50% - 65%, es ist Zeit den Wechselrichter neu zu starten.

Das hat sich geklärt, lag daran, dass aus der Batterie 2 Ausgänge rausgehen. Der eine war durch auf bypass gestellt, der andere hat die Batterie aufgeladen. Da der WR also nur an einem Eingang was bekam, wurde nur 50% Effizient angezeigt.
Das  neue Problem ist aber, dass die Bypasswerte vom PowerZero App (für die Bluepalm Batterie) nicht mit denen Werten des OpenDTUoB passen. Die Werte sind meistens auch die Hälfte (App zB 240W / OpenDTUoB 120W/Effizienz 95%). Schuss Electronic, die den Speicher über Hornbach vertreiben, haben mir aber eine neue Firmware für den Speicher freigegeben, da soll ich mal schauen, ob die Werte besser werden. Tja, so wird man zum BetaTester. 

>Deine Spannungen kann ich auch nicht nachvollziehen. Mein HM-600 hängt an 2 Paneelen mit je 144halbzellen. Selbst bei <10W pro Eingang hab ich noch >30V Spannung. Warum du also als Start und Stopp Spannung 20/19 eintragen musst, erschließt sich mir nicht. Die echte PV Spannung ist recht schnell >30V und bleibt da, selbst bis 5W runter, eigentlich dauerhaft weit drüber.

habe jetzt mal 30/29 eingetragen. Das funktioniert problemlos, wenn die Batterie ins Stromnetz einspeisst. Tagsüber, wenn der bypass drin ist (also nur ein Panel direkt zum WR geht - der zweite Eingang geht zum speichern) gibts immer wieder Aussetzer (Durch Fernsteuerung abgeschaltet). Vorher gabs sogar FB Überstrom. Da werd ich etwas nervös. 

![Bildschirmfoto 2024-03-31 um 18 38 08](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/a7a9ac3f-dd0e-4c66-b0e9-749c722122a2)

links gehen jeweils beide Panels rein, dann durch 2 MPPTs rechts dann in den WR raus.

![IMG_6826](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/18442198/5c1e6205-d8aa-4fc3-a056-d1a5d2ea4fc1)

**Reply by spcqike** [reply]
> Da der WR also nur an einem Eingang was bekam, wurde nur 50% Effizient angezeigt.

das halte ich für ein Gerücht. Die Effizienz ist nicht das Ergebnis der belegten Eingänge. Auch wenn nur an einem Eingang bspw 105W DC rein gehen und der wechselrichter damit 100W AC liefert, ergibt das eine Effizienz von 95%. 

Wenn er eine so niedrige Effizienz hat, regelt er unsauber. Dann macht er aus 105W DC bspw 50W AC seitig. Das passiert wenn die Spannung ungünstig war oder ist und er nicht richtig regeln kann. So zumindest die hier beschriebenen Erfahrungen anderer User.

**Reply by Manos1966** [reply]
Fehler 46 FlyBack Überspannung  & 47 FlyBack Überström !
sie kommen mir bekannt vor....

**Reply by Manos1966** [reply]
Ja, da war das hier
https://github.com/helgeerbe/OpenDTU-OnBattery/issues/613
es ging um eine 25V Batterie, aber deine ist 48V, ich weiss nicht ob es hilft.
